packages/lit-auth-client/src/lib/providers/EthWalletProvider.ts

```diff
@@ -168,10 +168,17 @@ export default class EthWalletProvider extends BaseProvider {
         address: address,
       };
     } else {
-      authSig = await checkAndSignAuthMessage({
-        chain,
-        nonce: litNodeClient.latestBlockhash!,
-      });
+      // only allow calls if we are in a browser as it assumes browser extension wallets are used.
+      if (isBrowser()) {
+        authSig = await checkAndSignAuthMessage({
+          chain,
+          nonce: nonce,
+        });
+      } else {
+        throw new Error(
+          "Could not find a 'signMessage' implementation on the provided signer instance"
```

Comment:
Just noting it for future reference that we need a blockchain client interface soon to support anything other than ethers. eg. viem